operations/utility_storage/Ceph_upgrade_tool_Usage.md

```diff
@@ -0,0 +1,81 @@
+# `ceph-upgrade-tool.py` Usage
+
+* [Introduction](#introduction)
+* [Usage](#usage)
+* [What the `ceph-upgrade-tool.py` does](#what-the-ceph-upgrade-toolpy-does)
```

Comment:
Easier to read, would obviously need to adjust the header to match
```suggestion
* [What is `ceph-upgrade-tool.py`](#what-is-ceph-upgrade-toolpy)
```

Reply:
Updated